README.md

```diff
@@ -10,10 +10,34 @@ It's totally not the first approach of this kind. But: this one should be easily
 
 Currently, Wireshell  consists of the following basic commands:
 
-#### New
+#### Fresh installation
 
 ```
-    $ wireshell new
+$ wireshell new /path/where/to/install --dbUser=db-user --dbName=db-name --dbPass=db-password --httpHosts=pw.dev --adminUrl=processwire --username=admin --userpass=abcd1234 --useremail=someone@example.com
+```
+
+If you don't pass the values, it will ask interactively.
+
+#### Profile installation
+
+```
+$ wireshell one-click-install /path/where/to/install --dbUser=db-user --dbName=db-name --dbPass=db-password --httpHosts=pw.dev --adminUrl=processwire --username=admin --userpass=abcd1234 --useremail=someone@example.com --profile=/path/to.zip
+```
+
+You can also install profiles. Current structure of zip is as
+
+```
+myprofile/
+    site-default/
+        modules
+        templates
+    composer.json
+```
```

Comment:
That is if there is a composer.json install the dependencies from it. You can see in the Installer command it tries to execute the `composer install` via symfony/process.